README.md

```diff
@@ -3,406 +3,228 @@
-
-The following fixture contains a simple test that types a developer name into a text editor and then clicks the Submit button.
-
-```js
-import { Selector } from 'testcafe';
-
-fixture `Getting Started`
-    .page `https://devexpress.github.io/testcafe/example`;
-
-test('My first test', async t => {
-    await t
-        .typeText('#developer-name', 'John Smith')
-        .click('#submit-button');
-});
-```
-
-All test actions are implemented as async functions of the [test controller object](https://devexpress.github.io/testcafe/documentation/test-api/test-code-structure.html#test-controller) `t`.
-This object is used to access test run API.
-To wait for actions to complete, use the `await` keyword when calling these actions or action chains.
-
-#### Observing Page State
-
-TestCafe allows you to observe the page state.
-For this purpose, it offers special kinds of functions that will execute your code on the client:
-[Selector](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html) used to get direct access to DOM elements
-and [ClientFunction](https://devexpress.github.io/testcafe/documentation/test-api/obtaining-data-from-the-client.html) used to obtain arbitrary data from the client side.
-You call these functions as regular async functions, that is you can obtain their results and use parameters to pass data to them.
-
-The selector API provides methods and properties to select elements on the page and get theirs state.
-
-For example, clicking the Submit button on the sample web page opens a "Thank you" page.
-To get access to DOM elements on the opened page, the `Selector` function can be used.
-The following example demonstrates how to access the article header element and obtain its actual text.
-
-```js
-import { Selector } from 'testcafe';
-
-fixture `Getting Started`
-    .page `http://devexpress.github.io/testcafe/example`;
-
-test('My first test', async t => {
-    await t
-        .typeText('#developer-name', 'John Smith')
-        .click('#submit-button');
-
-    const articleHeader = await Selector('.result-content').find('h1');
-
-    // Obtain the text of the article header
-    let headerText = await articleHeader.innerText;
-});
-```
-
-For more information, see [Selecting Page Elements](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/).
-
-#### Assertions
-
-A functional test also should check the result of actions performed.
-For example, the article header on the "Thank you" page should address a user by the entered name.
-To check if the header is correct, you have to add an assertion to the test.
-
-The following test demonstrates how to use [build-in assertions](https://devexpress.github.io/testcafe/documentation/test-api/assertions/).
-
-```js
-import { Selector } from 'testcafe';
+Read the [Getting Started](https://devexpress.github.io/testcafe/documentation/getting-started/) page for a more detailed guide.
```

Comment:
Lack of formatting here, I suppose it needs some padding between gif and this text or we can highlight this sentence somehow. See
https://github.com/VasilyStrelyaev/testcafe/blob/1c522c487fcd7b5a28ba33415ac1cf0a0e1fbae1/README.md#viewing-the-results
It doesn't look neatly.